packages/firestore/src/core/event_manager.ts

```diff
@@ -51,7 +51,7 @@ export class EventManager implements SyncEngineListener {
     q.canonicalId()
   );
 
-  private onlineState: OnlineState = 'Unknown';
+  private onlineState: OnlineState = OnlineState.Unknown;
```

Comment:
You can now probably move the explicit type here (i.e. restore it back to)

```ts
private onlineState = OnlineState.Unknown;
```

?

Reply:
Done. Note that this was part of a straight revert.